spec/views/events/_apply_button_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe "events/_apply_button", type: :view do
+  before(:each) do
+    @event = assign(:in_application_phase, FactoryGirl.create(:event))
```

Comment:
I think you misunderstood me.
It was `assign(:event, FactoryGirl.create(:event))` before, and what i meant was something like `assign(:event,  FactoryGirl.create(:event, :in_application_phase)`, in order to give the event another property. The way it is now, the instance var in_application_phase is set, and thats not what we want :)
Also, I missed at the first look that this is a test for a partial. And since the @event is directly passed to the partial in the render call, we can skip the assign()-part.
So, after all this line should be `@event = FactoryGirl.create(:event, :in_application_phase)`